### PR TITLE
fix deprecation warning

### PR DESCRIPTION
**Pull request by mikeferguson**

### CODE REVIEW

**Comment by mikeferguson** [comment]
Tested on Foxy with a ust-20lx

**Comment by mikeferguson** [comment]
This won't work on Dashing, so we should probably merge #71 and do a new/final release to dashing before merging this

File: src/urg_node.cpp [hunk 1]
```diff
@@ -116,7 +116,7 @@ void UrgNode::initSetup()
 
   diagnostic_updater_.add("Hardware Status", this, &UrgNode::populateDiagnosticsStatus);
 
-  this->set_on_parameters_set_callback(std::bind(&UrgNode::param_change_callback, this,
+  this->add_on_set_parameters_callback(std::bind(&UrgNode::param_change_callback, this,
```
Comment (jacobperron) [inline src/urg_node.cpp]: I think we need to hold a reference to the returned shared pointer, otherwise the callback may be unregistered.

**Review by jacobperron — COMMENTED** [review]
Now that deprecated API has been removed, releasing this patch into `rolling` should fix http://build.ros2.org/view/Rbin_uF64/job/Rbin_uF64__urg_node__ubuntu_focal_amd64__binary/17/

**Comment by mikeferguson** [comment]
passes, has crustify issues on F/R - which I'll fix in separate PR

**Comment by Yu1107** [comment]
same problem on ubuntu 18.04 dashing

**Pull request merged**